spyder/widgets/tests/test_switcher.py

```diff
@@ -102,3 +103,16 @@ def test_switcher_filter_and_mode(dlg_switcher, qtbot):
     edit.setText(":")
     qtbot.wait(1000)
     assert dlg_switcher.count() == 1
+
+
+def test_switcher_filter_unicode(dlg_switcher, qtbot):
+    """Test filter with unicode."""
+    edit = dlg_switcher.edit
+
+    # Initially cvs mode with five rows
+    assert dlg_switcher.count() == 6
+    dlg_switcher.show()
+    # Match one row by name
+    edit.setText('uní')
```

Comment:
I think the test would be better if among the switcher entries would be text with non-ascii (e.g. Chinese or Arabic) characters, and at this point we filter some of them.